discovery/gossiper.go

```diff
@@ -922,9 +926,10 @@ func (d *AuthenticatedGossiper) networkHandler() {
 			// First, we'll now create new fully signed updates for
 			// the affected channels and also update the underlying
 			// graph with the new state.
-			chanPolicies, newChanUpdates, err := d.processChanPolicyUpdate(
-				policyUpdate,
-			)
+			chanPolicies, newChanUpdates, err :=
```

Comment:
Style nit: imo overflowing on one line is better than making a break and created a "dangling" function invocation.

Reply:
Wasn't sure how hard our line limit is. But overflow looks better indeed.